tests/unit/test_pynblint.py

```diff
@@ -192,3 +192,11 @@ def test_get_duplicate_notebooks(test_input, expected, repos):
 ])
 def test_get_untitled_notebooks(test_input, expected, repos):
     assert repo_linting.get_untitled_notebooks(repos[test_input]) == expected
+
+
+@pytest.mark.parametrize("test_input,expected", [
+    ("DuplicatesNoUntitled", True),
+    ("UntitledNoDuplicates", True)
```

Comment:
What's the point of testing the same function outcome twice here? You should also test the `False` outcome (to discover that it is never returned, as in Python an empty list is not `None`).